tests/model/templates/team-response-template.model.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { TeamResponseTemplate } from '../../../src/app/core/models/templates/team-response-template.model';
+import { EMPTY_BODY_GITHUB_COMMENT } from '../../constants/githubcomment.constants';
+
+const EXPECTED_TEAM_RESPONSE_HEADER = "# Team's Response";
+const EXPECTED_TEAM_RESPONSE_TEMPLATE_CONTENT = 'This is a simple response';
+const DUPLICATE_ISSUE_NUMBER = 100;
+
+const TEAM_RESPONSE_WITH_WHITESPACE =
+  "# Team's Response \r\n " + '  \n This is a simple response   \r\n \n\n ' + '## Duplicate status (if any): --';
```

Comment:
Prettier auto formats the double quotes to start on a new line so I'm unable to shift it to be on the same line as the variable 😓